src/qt/bitcoin.cpp

```diff
@@ -5,9 +5,7 @@
 #if defined(HAVE_CONFIG_H)
 #include <config/bitcoin-config.h>
 #endif
-
 #include <qt/bitcoingui.h>
-
```

Comment:
Please don't remove unrelated whitespace.

Reply:
that's not funny

Reply:
It wasn't meant to be funny...?

Reply:
@alexeyneu you should only commit what is relevant to the PR.

Reply:
who you mean by developer?And what about  this pair of your tools , does its build results matter? And one of you two for sure should know do you provide vista support?

Reply:
@alexeyneu https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md

Reply:
This document gives a clue about the expected compatibility: https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes.md#compatibility

Reply:
so no vista